lib/talon.ex

```diff
@@ -52,11 +52,15 @@ defmodule Talon do
     repo = opts[:repo]
 
     quote location: :keep do
-      @__resources__  Application.get_env(:talon, :resources, [])
+      @__resources__  Application.get_env(:talon, :resources, []) # TODO: make easier to inject/test (DJS)
+
+      @__dashboard__  Application.get_env(:talon, :dashboard, nil) # TODO: support more than one dashboard (DJS)
```

Comment:
I agree, see below - I had a TODO on `@__page_map__` for this (with "page" incorrectly singular). I still wasn't certain if we shouldn't maintain a list of pages that are dashboards, as they get specific treatment when presented (i.e., listed first, before resources). Other pages may occur after resources. 

Note, I was leaning towards the idea of supporting multiple dashboards, if desired. 

Take look here, for example: https://adminlte.io/themes/AdminLTE/index2.html

Instead of first-class handling of  dashboards, this could be handled with pages and a concept of page_groups, where `dashboards` are one such pre-defined group. In the layout, the user can then decide where to put a given page group in the sidebar menu (e.g., dashboard group, resources, other page groups)

We could have simple default heuristic for presenting page groups (see example AdminLTE above):
- One item, just display page name without a container (current way we display dashboard link)
- Less than N, present as titled container of child page links, but always expanded
- More than N, present as collapsable container of children